kehadiran_scraper/main.py

```diff
@@ -1,93 +1,201 @@
 import asyncio
 import datetime as dt
 import logging
+from os import environ
 import os
+from random import random
 import typing as t
 
 import aiofiles
 import aiohttp
 
-# TODO: Read from env var for some of these
-SCRAPE_CHUNK_SIZE = 50
-SLEEP_S = 1
-FIRST_DATE = dt.date(1959, 9, 11)
+SCRAPE_CHUNK_SIZE = int(environ.get("SCRAPE_CHUNK_SIZE", 5))
+SLEEP_S = float(environ.get("SLEEP_S", 0.5))
+DISCONNECT_BACKOFF_S = float(environ.get("DISCONNECT_BACKOFF_S", 10))
+MIN_DATE = dt.datetime.strptime(
+    environ.get("MIN_DATE", "1959-09-11"), "%Y-%m-%d"
+).date()
+MAX_DATE_DEFAULT = dt.datetime.strftime(dt.datetime.today(), "%Y-%m-%d")
+MAX_DATE = dt.datetime.strptime(
+    environ.get("MAX_DATE", MAX_DATE_DEFAULT), "%Y-%m-%d"
+).date()
 URL_PATTERN = "https://www.parlimen.gov.my/files/hindex/pdf/DR-{day}{month}{year}.pdf"
-TEST_OUTPUT_DIR = ".test.out/"
+OUTPUT_DIR = "output/"
```

Comment:
I think we should make this configurable via env variable too & don't include .pdf in the commit. In production, we would be setting `OUTPUT_DIR` as the `kehadiran` directory. This means the cron in prod doesn't have to copy the files & avoid storing duplicate PDF files in prod machine.

Reply:
I seem, makes sense. I initially thought we wanted to share the pdf files to the public as well. But I guess in any case that's not the main goal (Main goal is the kehadiran json files). I'll make it configurable and not include `*.pdf` files

Reply:
@cglotr Done, please latest commit

Reply:
To clarify, we will also be sharing the PDFs in https://github.com/arikama/kehadiran. We will set the scraper to save the PDFs directly to this repo where it will be pushed to Github by the cron.

Reply:
@imranariffin

Reply:
Thanks, now I remember our discussion